### ISSUE TITLE
Improvement suggestions

### ISSUE THREAD

**Issue by micobarac** [ngstack/translate#13]
I've decided to drop my `@ngx-translate` implementation and turn it into `@ngstack/translate` implementation. Along the way, I've hit some obstacles, with `@ngstack/translate`  missing some basic features `@ngx-translate` has. So, I suggest the following improvements:

1. Translate params should accept numbers, along with strings.

```typescript
export interface TranslateParams {
    [key: string]: string;
}
```

should be transformed into:

```typescript
export interface TranslateParams {
    [key: string]: string | number;
}
```

2. `get` method should translate array of strings, along with string:

```typescript
get(key: string, params?: TranslateParams, lang?: string): string;
```

should be transformed into:

```typescript
get(key: string | string[], params?: TranslateParams, lang?: string): string | string[];
```

Keep up the good work! :D

**Comment by DenysVuika** [comment]
Very good suggestion, thanks, @micobarac

**Comment by DenysVuika** [comment]
What is the case for having multiple input strings and single interpolation object, compared to just calling the `get` method multiple times or using it with array.map ? The `get` is sync, compared to ngx/translate that has it as observable.

**Comment by micobarac** [comment]
I am using Angular Material data table with translatable paginator for which I need to translate a dosen keys at once. That was the reason to possible incorporate this option inside get method.

**Comment by DenysVuika** [comment]
Makes sense, thanks.

**Comment by micobarac** [comment]
`MatPaginatorIntl`

[https://material.angular.io/components/paginator/api](https://material.angular.io/components/paginator/api)

**Comment by DenysVuika** [comment]
Implemented and will be available shortly with 1.2 milestone
